pages/docs/reference/building-mpp-with-gradle.md

```diff
@@ -962,7 +962,7 @@ kotlin {
 </div>
 </div>
 
-Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#experimental-metadata-publishing-mode) and the project 
+Likewise, if a multiplatform library is published in the experimental [Gradle metadata publishing mode](#metadata-publishing) and the project 
```

Comment:
Remove `experimental` from this line, too? Or even rephrase it as `if a multiplatform library is [published with Gradle module metadata](link)`?